features/tools.feature

```diff
@@ -4,38 +4,8 @@ Feature: New Kedro project with tools
     Given I have prepared a config file with tools "none"
     When I run a non-interactive kedro new without starter
     Then the expected tool directories and files should be created with "none"
-    Given I have installed the project dependencies
-    When I execute the kedro command "run"
-    Then I should get a successful exit code
 
   Scenario: Create a new Kedro project with all tools except 'viz' and 'pyspark'
     Given I have prepared a config file with tools "lint, test, log, docs, data"
     When I run a non-interactive kedro new without starter
     Then the expected tool directories and files should be created with "lint, test, log, docs, data"
```

Comment:
Is there any reason to keep this bit?

Reply:
I guess these are just testing the templates instead of any of the starters this will just be a sanity check.

Reply:
Would it make sense to then still have the check for exit code?

Reply:
You are right, added. 👍

Reply:
Oh looks like that failed there isn't a 'result' I think the assertions should fine.